doc/source/mne_report_tutorial.rst

```diff
@@ -45,6 +45,12 @@ To properly render `trans` and `covariance` files, add the measurement informati
     mne report --path MNE-sample-data/ --info MNE-sample-data/MEG/sample/sample_audvis-ave.fif \ 
         --subject sample --subjects-dir MNE-sample-data/subjects --verbose
 
+To render whitened `evoked` files, add the noise covariance file::
+    
+    mne report --path MNE-sample-data/ --info MNE-sample-data/MEG/sample/sample_audvis-ave.fif \ 
+        --cov MNE-sample-data/MEG/sample/sample_audvis-cov.fif --subject sample \
+        --subjects-dir MNE-sample-data/subjects --verbose
```

Comment:
Yeah I think it needs it:

![screenshot from 2015-04-27 10 40 13](https://cloud.githubusercontent.com/assets/2365790/7353624/d8e8ab5a-ecc9-11e4-8072-6a8e1a05deee.png)